pkg/api/vlabs/validate.go

```diff
@@ -274,14 +277,21 @@ func (a *Properties) Validate() error {
 	return nil
 }
 
-func (a *Properties) ValidateNetworkPolicy() error {
-
+func (a *Properties) validateNetworkPolicy() error {
 	if a.OrchestratorProfile.OrchestratorType != Kubernetes {
 		return nil
 	}
 
 	networkPolicy := a.OrchestratorProfile.KubernetesConfig.NetworkPolicy
```

Comment:
Sure, agreed. I'll add a check in OrchestratorProfile.Validate() to cover both me and Calico.